custom.d.ts

```diff
@@ -1,4 +1,6 @@
 declare module "*.svg" {
-  const content: React.FunctionComponent<React.SVGAttributes<SVGElement>>;
-  export default content;
+  import React = require("react");
+  export const ReactComponent: React.FC<React.SVGProps<SVGSVGElement>>;
+  const src: string;
+  export default src;
 }
```

Comment:
Try to slightly change exports, make `const ReactComponent: React.FC<React.SVGProps<SVGSVGElement>>;` a default export, then, you'll be able to use pretty imports like
`import MenuLogo from "../../assets/logos/menuLogo.svg";`
I hope)